pilot/pkg/networking/core/accesslog.go

```diff
@@ -49,7 +49,8 @@ var (
 	// We need to propagate these as part of access log service stream
 	// Logging them by default on the console may be an issue as the base64 encoded string is bound to be a big one.
 	// But end users can certainly configure it on their own via the meshConfig using the %FILTER_STATE% macro.
-	envoyWasmStateToLog = []string{"wasm.upstream_peer", "wasm.upstream_peer_id", "wasm.downstream_peer", "wasm.downstream_peer_id"}
+	envoyWasmStateToLog = []string{"upstream_peer", "downstream_peer", // start from 1.24.0
```

Comment:
Accesslog has two configure path? one from telemetry api and one here. Why not check proxy version here?

Reply:
this's the tradditional way, hard to check version here.